eng/Generate.ps1

```diff
@@ -382,6 +383,13 @@ if (!$noBuild) {
     Invoke-TypeSpecSetup
 }
 
+$localTestNugetSourceCreated = $false
+$localTestNugetWorkFolder = Join-Path ([System.IO.Path]::GetTempPath()) ([string][System.Guid]::NewGuid())
+# Only MgmtCustomizations project needs the LocalNugetServer now
+if($keys -contains "MgmtCustomizations"){
```

Comment:
How about keeping `BaselineContract` folder and put the corresponding `.nupkg` files under the corresponding folder by name convention?
And then, we can check if the folder and package file exists for the current test project instead of hard-coded name.